V2

Eeshau opened the pull request.

Divide-By-0 commented:
Fix merge conflict, add padding at the bottom,  change partner button to say fill form, add me so I can view responses on typeform!

Metachaser24 commented:
Fixed conflict, added padding, changed button text. Changed from typeform to google form and added your email

Divide-By-0 commented:
the url still seems to be to the typeform? @Metachaser24

wryonik committed:
Initialize next website

wryonik committed:
cp2

wryonik committed:
Hero Section

wryonik committed:
Partners and Porject Section

wryonik committed:
chore: Add noise background and update images

Eeshau committed:
Footer

Eeshau committed:
FAQ section

Eeshau committed:
resources section

wryonik commented on src/components/Footer.tsx:
```diff
@@ -1,5 +1,58 @@
+        {/* ICON LINKS */}
+        <div className="flex flex-row gap-3 ml-4">
+          <a href='/https://x.com/zkemail?lang=en'>
+            <svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16" fill="none">
```
Do not add svgs directly into the files. It will clutter them with unnecessary things

wryonik commented on src/sections/FAQs.tsx:
```diff
@@ -1,5 +1,101 @@
+    <div className="w-full mx-auto mt-10 z-10 px-8 sm:px-36">
+      <div className='flex flex-row justify-between'>
+        <div>
+          <h1 className="text-3xl font-semibold mb-6 h3 text-left " style={{textAlign:'left'}}>FAQs</h1>
```
Why are you repeating textAlign property in both tailwind and CSS in JSX? Remove it from one place else it can cause confusions while trying to change this property in the future

wryonik commented on src/sections/FAQs.tsx:
```diff
@@ -1,5 +1,101 @@
+      <div className='flex flex-row justify-between'>
+        <div>
+          <h1 className="text-3xl font-semibold mb-6 h3 text-left " style={{textAlign:'left'}}>FAQs</h1>
+          <p className="text-left mb-4 text-[#A8A8A8]">
```
Try not to hardcode colors like this. I've defined some colors in the CSS file. If you want to use Tailwind, you can define these colors in the Tailwind config and use them from there. This way, whenever we want to change a color, we just need to update it in the config file, and all the colors will be automatically updated

wryonik commented on src/sections/FAQs.tsx:
```diff
@@ -1,5 +1,101 @@
+          <a  href='https://zkemail.gitbook.io/zk-email' target='_blank'>
+            <button className='flex flex-row justify-between gap-1 items-center px-[12px] py-[8px] bg-[#1C1C1C] border-solid border-2 border-[#3B3B3B] rounded-[12px] text-[#F5F3EF] text-[16px] whitespace-nowrap'>
+              Read Docs
+              <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14" fill="none">
```
It seems like you are using the same svg file many times but pasted the whole content everywhere (like this right arrow sign svg). It would be better if you can define it in one file and use it everywhere else

wryonik commented on src/sections/FAQs.tsx:
```diff
@@ -1,5 +1,101 @@
+            >
+              <div className='flex gap-3 items-center'>
+                {/* blue diagonal icon  */}
+                <div className="w-3 h-3 rotate-[-45deg] rounded-[2.268px] border border-[#2962A5] bg-[#062E5D]"></div>
```
Round values like 2.268px to the nearest decimal place, as they will be easier to handle

wryonik commented on src/sections/FAQs.tsx:
```diff
@@ -60,7 +60,7 @@ const Faq = () => {
               <div className='flex gap-3 items-center'>
-                {/* blue diagonal icon  */}
+                {/* blue diagonal diamond icon  */}
                 <div className="w-3 h-3 rotate-[-45deg] rounded-[2.268px] border border-[#2962A5] bg-[#062E5D]"></div>
```
Since this is a repeating component, you can create a separate component from it and use it everywhere. This approach makes it easier to understand which component is being used, and you won't have to copy this complex line (with lots of colors and other elements) everywhere

wryonik commented on src/sections/Resources.tsx:
```diff
@@ -1,5 +1,298 @@
+            <button
+              onClick={previous}
+              disabled={currentIndex === 0}
+              className="flex flex-row justify-between items-center gap-1 border-solid border-[2px] border-[#3B3B3B] rounded-[12px] px-[16px] py-[5px] bg-[#1C1C1C] font-semibold"
```
Damn, this is a lot of CSS. Can you make a common component for this as well as this is also repeating a lot. In this way, you just have to pass `onClick`, `label`, `icon`, `icon-position`, `color` and maybe a few other props only

wryonik commented on src/sections/Resources.tsx:
```diff
@@ -0,0 +1,298 @@
+  const visibleCards = getVisibleCards();
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const next = () => {
```
rename to `handleNext` or something which defines action as well

wryonik committed:
chore: Fix projects cards border

wryonik committed:
chore: Update how zkemail works section

wryonik committed:
ui: responsiveness changes in dektop view

wryonik committed:
chore: Fix footer UI

wryonik committed:
chore: Add transitions in FAQ section and fix section widths

wryonik committed:
ui: Fix UI of faq section

wryonik committed:
ui: Update hero image

wryonik committed:
chore: Copy changes

wryonik committed:
ui: Add hover animations on projects cards

wryonik committed:
ui: Add hover animations on zkemail working cards

wryonik committed:
ui: Update mobile design for home page

wryonik committed:
ui: update button components

wryonik committed:
wip: update content and blogs page

wryonik committed:
Migrate to next and add blogs page

wryonik committed:
UI: Add changelogs page design

wryonik committed:
UI: Update footer links

wryonik committed:
UI: Review changes

wryonik committed:
UI: Add projects page

wryonik committed:
chore: update accessibility related things

wryonik committed:
ui: Update footer icons

wryonik committed:
ui: Performance optimisations

wryonik committed:
ui: import marquee component dynamically

wryonik committed:
ui: update navbar component

wryonik committed:
ui: Update mobile navbar

wryonik committed:
ui: Add privacy policy page

wryonik committed:
ui: Update image sizes

wryonik committed:
ui: Update links

wryonik committed:
chore: update assets

wryonik committed:
ui: update navbar height

wryonik committed:
chore: performance updates

wryonik committed:
fix: Review fixes

wryonik committed:
ui: Add background in navbar

wryonik committed:
ui: Review fixes

- Remove scrollbar from resources cards
- Add learn more in projects cards

wryonik committed:
ui: Fix navbar animation

wryonik committed:
ui: Add blue and fade animations

wryonik committed:
ui: Add animations in the home page

wryonik committed:
ui: Add animations in all pages

wryonik committed:
chore: Fix eslint errors

wryonik committed:
ui: Add animations in blogs page

wryonik committed:
chore: copy change

wryonik committed:
chore: Update metadata and favicon

wryonik committed:
chore: ui updates

- Update link
- update link color on hover
- Add loader
- performance improvements

wryonik committed:
:chore: Fix blog rendering in some pages

wryonik committed:
:chore: fix max width for the landing page

wryonik committed:
:chore: update iyk logo

PrakharSingh0908 committed:
UI changes with Shubham

wryonik committed:
:chore: update project icons and copy changes

saugardev committed:
docs: add most recent changelogs

wryonik committed:
:chore: Fix build error

wryonik committed:
:chore: Update changelogs

wryonik committed:
:chore: Rebase update
